lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -134,3 +164,14 @@ DetailsRenderer.ListDetailsJSON; // eslint-disable-line no-unused-expressions
  * }}
  */
 DetailsRenderer.CardsDetailsJSON; // eslint-disable-line no-unused-expressions
+
+/**
+ * @typedef {{
+ *     type: string,
+ *     text: string,
```

Comment:
It's the text for the code snippet. For this audit, they're the console message text. We're just that text in `<pre>` so it looks like DT's log output.